### PR TITLE
QE: Diverge product sync for SLES15 SP4

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

We caught the issue regarding the autoselection of the Server Applications Module.
The root cause was in the previous feature, where we are synchronizing via mgr-sync command the channel of the SUMA client tools. That action, automatically synchronize not only the client tools but all the inverted tree of dependencies for that channel, synchronizing its parents. So, when we tried to select the product in the next feature, in fact the selection was disabled, and so it prevented the automatically selection of the unrelated sub-product of SLES15 SP4 on regard of SUMA client tools, but at same time, it was one of the recommended products and so the scenario failed.

To fix it, we removed the scenario synchronizing the client tools in the previous feature, and rely only in the synchronization made on the feature adding SLES 15 SP4 product and its recommended sub-products.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- Cucumber tests were modified

- [x] **DONE**

## Links

Port:
- Manager-4.3

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by vandabarata — APPROVED** [review]

File: testsuite/features/reposync/srv_sync_channels.feature [hunk 1]
```diff
@@ -48,11 +48,6 @@ Feature: Be able to list available channels and enable them
     And I should see "Total time:" in the output
     And I should see "Repo URL:" in the output
 
-@susemanager
-  Scenario: Enable SLES15 SP4 SUMA client tools for creating bootstrap repositories
```
Comment (Bischoff) [inline testsuite/features/reposync/srv_sync_channels.feature:52]: we write elsewhere `SLES 15 SP4`

**Review by Bischoff — APPROVED** [review]
just a nitpick please

File: testsuite/features/reposync/srv_sync_channels.feature (hunk 1 above)
Comment (srbarrios) [reply testsuite/features/reposync/srv_sync_channels.feature:52]: Ok, but as you can see this is code we are removing.

Comment (Bischoff) [reply testsuite/features/reposync/srv_sync_channels.feature:52]: "oops"

**Pull request merged**